js/main.js

```diff
@@ -18,7 +31,7 @@ CRM.$(function($) {
         }
       });
       $('.price-set-option-content input').change( function () {
-        if ($(this).attr('data-amount').replace(/[^\/\d]/g,'') == 0 ) {
+        if (discountCalculateTotalFee() == 0) {
```

Comment:
@alifrumin Is there a reason we can't use the existing function rather than copying it here? Something like: https://github.com/mattwire/com.drastikbydesign.stripe/blob/5.x/js/civicrm_stripe.js#L274